tools/load-test/k8syaml/function.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: kubeless.io/v1beta1
+kind: Function
+metadata:
```

Comment:
souldn't namespace of the Function point to `load-test`?

Reply:
Yes, you may put the namespace in the metadata. Here, we wanted to control it from the gocode as there are a lot of `kubectl` usages with namespace information.